packages/flutter/test/material/checkbox_test.dart

```diff
@@ -139,6 +139,24 @@ void main() {
       hasEnabledState: true,
       isChecked: true,
     ));
+
+    await tester.pumpWidget(Theme(
+      data: theme,
+      child: const Material(
+        child: Checkbox(
+          value: null,
+          tristate: true,
+          onChanged: null,
+        ),
+      ),
+    ));
```

Comment:
Should we add tests for `value: true, tristate: true` and `value: false, tristate: true`?